scripts/install-mender.sh

```diff
@@ -151,7 +151,7 @@ add_repo() {
 
     if ! grep -F "$repo" /etc/apt/sources.list.d/mender.list; then
         echo "adding $repo to /etc/apt/sources.list.d/mender.list"
-        echo "$repo" >> /etc/apt/sources.list.d/mender.list
+        echo "$repo" > /etc/apt/sources.list.d/mender.list
     fi
 }
```

Comment:
I meant to substitute this whole `if` block with only: `echo "$repo" > /etc/apt/sources.list.d/mender.list`.

Reply:
(for some reason GitHub does not let me put this as a "suggestion")